src/App.js

```diff
@@ -1,13 +1,64 @@
 import React from "react";
-
+import Heading from "./Heading";
 import Bookings from "./Bookings";
 import "./App.css";
+import TouristInfoCards from "./TouristInfoCards";
+import Footer from "./Footer";
+import Restaurant from "./Restaurant";
+
+const info = [
+  "123 Fake Street, London, E1 4UD",
+  "hello@fakehotel.com",
+  "0123 456789"
+];
+
+const TouristInfoDetails = [
+  {
+    name: "Glasgow",
+    url: "https://peoplemakeglasgow.com",
+    pic:
+      "https://static.independent.co.uk/s3fs-public/thumbnails/image/2018/05/04/16/glasgow-main.jpg?quality=75&width=1368&auto=webp",
+    description:
+      "Glasgow is one of Europe's most vibrant and cosmopolitan cities. It boasts world famous art collections, the best shopping in the UK outside of London, sports and leisure facilities, an array of restaurants and bars, and the most vibrant and exciting nightlife."
+  },
+  {
+    name: "London",
+    url: "https://visitlondon.com",
+    pic:
+      "https://www.fodors.com/assets/destinations/2869/tower-bridge-london-england_980x650.jpg",
+    description:
+      "London, city, capital of the United Kingdom. It is among the oldest of the world's great cities—its history spanning nearly two millennia—and one of the most cosmopolitan. By far Britain's largest metropolis, it is also the country's economic, transportation, and cultural centre"
+  },
+  {
+    name: "Manchester",
+    url: "https://www.visitmanchester.com",
+    pic:
+      "https://dynamic-media-cdn.tripadvisor.com/media/photo-o/15/a1/ee/e8/the-famous-manchester.jpg?w=900&h=-1&s=1",
+    description:
+      "Manchester was right at the heart of the Revolution, becoming the UK's leading producer of cotton and textiles. Its famous for being the first industrialised city in the world and was responsible for the first ever working canal and the first ever railway line."
+  }
+];
 
 const App = () => {
   return (
     <div className="App">
-      <header className="App-header">CYF Hotel</header>
+      <Heading />
+      <div className="card-content">
+        {TouristInfoDetails.map((detail, index) => {
+          return (
+            <TouristInfoCards
+              key={index}
```

Comment:
Its best to try and avoid using the array index as the key here. Have a read of the docs here: https://legacy.reactjs.org/docs/lists-and-keys.html#keys
That can explain it better than I can! 😄